runtime/tests/runtime_integration_tests/test_sign_with_export_ecdsa.rs

```diff
@@ -157,3 +171,211 @@ fn test_sign_with_exported_never_derived() {
         )
     );
 }
+
+#[test]
+fn test_sign_with_exported_cdi_measurement_update_duplicate_cdi() {
```

Comment:
nit: can we have a test case that covers the prior bug? I'm thinking

1. export
2. Sign & check signature
3. export again (should fail because we didn't revoke)
4. Sign again and make sure the original key from 1 still works

Reply:
This test has steps 1, 3 and 4. I'll add step 2 and it would match this.

`original_cdi_resp` carries the original export to the very end, and checks that the signature still matches the certificate and that the export-cdi handle is valid.